<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="1" failures="0" name="unittest.suite._ErrorHolder-20180207071530" skipped="0" tests="1" time="0.000">
	<testcase classname="setUpClass (__main__" name="lxcaSetupTest)" time="0.000">
		<error message="name 'lxcaip' is not defined" type="NameError">
<![CDATA[Traceback (most recent call last):
  File "C:\Python34\Scripts\eclipse-workspace\Scripts_LXCA\LXCA_Setup_test.py", line 15, in setUpClass
    inst.driver.get('https://'+lxcaip)
NameError: name 'lxcaip' is not defined
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
